applicationset/services/repo_service.go

```diff
@@ -31,15 +33,23 @@ type Repos interface {
 
 	// GetDirectories returns a list of directories (not files) within the target repo
 	GetDirectories(ctx context.Context, repoURL string, revision string) ([]string, error)
-}
 
-func NewArgoCDService(db db.ArgoDB, gitCredStore git.CredsStore, submoduleEnabled bool) Repos {
+	// Close any open connections
+	Close()
+}
 
+func NewArgoCDService(db db.ArgoDB, gitCredStore git.CredsStore, submoduleEnabled bool, repoClientset repoapiclient.Clientset) (Repos, error) {
+	closer, repoClient, err := repoClientset.NewRepoServerClient()
+	if err != nil {
+		return nil, err
+	}
 	return &argoCDService{
 		repositoriesDB:   db.(RepositoryDB),
 		storecreds:       gitCredStore,
```

Comment:
I don't think we need the storereds any longer.

Reply:
@rumstead thanks for making the rpc clientset changes just wanted to confirm that you haven't pushed the change to remove storecreds yet right?